### DISCUSSION TITLE
Generate navmesh file

### DISCUSSION THREAD

**Discussion by BeckUp-Media** [ppiastucki/recast4j#206] [Q&A]
![image](https://github.com/user-attachments/assets/0d5adc8e-ae33-421d-8550-57d7c8130b7e)
Hi, can anyone explain why i cant export my loaded .obj file as navmesh? Ive tried this, but no chance to get an navmesh file. I can only export an voxel file.

**Comment by BeckUp-Media** [comment]
ok, looks like i need to switch x,y,z array to x,zyz array. Good dawn, Blender has different values :D

**Comment by ppiastucki** [comment]
There is no functionality to save the generated navmesh. It might be added some day though. This tool is meant mostly for demo purposes not to be used as nav mesh generator / editor.

**Comment by BeckUp-Media** [comment]
yeah, but its nearly done with my export. :D biggest problem is that i need to swap something. I get negative coordinates ony my navmesh, bcs my .obj file was a bit curstom
